packages/bpk-docs/src/pages/NativeCardsPage/NativeCardsPage.js

```diff
@@ -155,17 +155,23 @@ const components = [
   },
 ];
 
+const isNeo = process.env.BPK_NEO;
+
 const NativeCardsPage = ({ ...rest }) => (
   <DocsPageBuilder
     title="Cards"
-    blurb={[
-      <Paragraph>
-        Cards are used to group related items within the UI. They allow complex
-        datasets to be broken down into individual, distinct areas for easy
-        consumption. Cards are used as a signifier to give affordance to a
-        clickable set of elements.
-      </Paragraph>,
-    ]}
+    blurb={
+      isNeo
+        ? []
+        : [
+            <Paragraph>
+              - Cards are used to group related items within the UI. They allow
```

Comment:
Did you copy pasta this from a diff, as there's `-` symbols interspersed in it 😆